Kickstarter-iOS/Locales/Base.lproj/Localizable.strings

```diff
@@ -109,6 +109,7 @@
 "Check_out_these_handpicked_projects" = "Check out these handpicked projects just for you.";
 "Check_your_inbox_to_complete_this_simple_step" = "Check your inbox to complete this simple step.";
 "Check_your_payment_details" = "Check your payment details";
+"Checkout" = "Checkout";
```

Comment:
Do you want to update `PostCampaignCheckoutViewController` to use this string instead of the hardcoded `Checkout` in this pr, since that pr is merged now?